documentation/en/user/source/configuration/storage.rst

```diff
@@ -272,6 +271,173 @@ GeoServer ``topp:states`` sample layer on a fictitious ``my-geowebcache-bucket``
+
+Property *eagerDelete* defines if we should remove a whole database file or instead perform a delete query. The property *useCreateTime* can be used to activate or deactivate the insertion of the tile creation time (see section above). Property *executorConcurrency* controls the parallelism used to perform certain operations, like the truncate operation for example. Property *mbtilesMetadataDirectory* defines the directory where we should look for user provided MBTiles metadata.
+
+Replace Operation
+``````````````````
+
+As said before, if the cache is running we cannot simply switch SQLite files, we need to make sure that all connections are closed. The replace operation was created for this propose. The replace operation will first copy the new file side by side the old one, then block the requests to the old file, close the connections tot he old file, delete the old one, rename the new file to current one, reopen the new db file and start serving requests again. Should be almost instant.
+
+A REST entry point for this operation is available, it will be possible to submit a ZIP file or a single file along with the request. We can also ask the replace operation to use an already present file or directory. When using a directory the directory structure will be used to find the destination of each file, all the paths will be assumed to be relative to the store root directory. This means that if we want to replace a store content with another store content (a seeded one for example) we only need to ZIP the second store root directory and send it as a replacement and the seeded store content will me merged with the first store content.
+
+There is four ways to invoke this operation. Follows an example of all those variants invocations using CURL.
+
+Replace a single file uploading the replacement file:
+
+.. code-block:: none
+
+  curl -u geowebcache:secured -H 'Content-Type: multipart/form-data'
+    -F "file=@tiles_0_0.sqlite"
+    -F "destination=EPSG_4326/sf_restricted/image_png/null/10/tiles_0_0.sqlite"
+    -F "layer=sf:restricted"
+    -XPOST 'http://localhost:8080/geowebcache/rest/sqlite/replace'
+
+Replace a single file using a file already present on the system:
+
+.. code-block:: none
+
+  curl -u geowebcache:secured -H 'Content-Type: multipart/form-data'
+    -F "source=/tmp/tiles_0_0.sqlite"
+    -F "destination=EPSG_4326/sf_restricted/image_png/null/10/tiles_0_0.sqlite"
+    -F "layer=sf:restricted"
+    -XPOST 'http://localhost:8080/geowebcache/rest/sqlite/replace'
+
+Replace multiple files uploading a ZIP file:
+
+.. code-block:: none
+
+  curl -u geowebcache:secured -H 'Content-Type: multipart/form-data'
+    -F "file=@tiles.zip"
+    -F "layer=sf:restricted"
+    -XPOST 'http://localhost:8080/geowebcache/rest/sqlite/replace'
+
+Replace multiple files using a directory already present on the system:
+
+.. code-block:: none
+
+  curl -u geowebcache:secured -H 'Content-Type: multipart/form-data'
+    -F "source=/tmp/tiles"
```

Comment:
Will it use all files available? Filter only by zip? Filter by expected layer name somehow?